### PR TITLE
Add RuntimeConfig datasource

### PR DESCRIPTION
**Pull request by upodroid**
Fixes: https://github.com/hashicorp/terraform-provider-google/issues/7428

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-datasource
`google_runtimeconfig_config`
```

### CODE REVIEW

File: third_party/terraform/website/docs/d/runtimeconfig_config.html.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* [API documentation](https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/v1beta1/projects.configs)
+* How-to Guides
+    * [Runtime Configurator Fundamentals](https://cloud.google.com/deployment-manager/runtime-configurator/)
+
+## Example Usage
+
+```hcl
+data "google_runtimeconfig_config" "run-service" {
+  name = "my-service"
+  description = "foobat"
```
Comment (c2thorn) [inline third_party/terraform/website/docs/d/runtimeconfig_config.html.markdown:23]:
```suggestion
  description = "foobar"
```
We could just remove `description` entirely right? It wouldn't help identify the resource

File: third_party/terraform/website/docs/d/runtimeconfig_config.html.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+* `name` - (Required) The name of the Cloud Run Service.
+
+- - -
+
+* `project` - (Optional) The project in which the resource belongs. If it
+    is not provided, the provider project is used.
+
+## Attributes Reference
+
+See [google_runtimeconfig_config](https://www.terraform.io/docs/providers/google/r/google_runtimeconfig_config.html#argument-reference) resource for details of the available attributes.
```
Comment (c2thorn) [inline third_party/terraform/website/docs/d/runtimeconfig_config.html.markdown:40]:
```suggestion
See [google_runtimeconfig_config](https://www.terraform.io/docs/providers/google/r/runtimeconfig_config.html#argument-reference) resource for details of the available attributes.
```

**Review by c2thorn — CHANGES_REQUESTED** [review]
seems solid, a couple of small asks

File: third_party/terraform/website/docs/d/runtimeconfig_config.html.markdown (hunk 1 above)
Comment (upodroid) [reply third_party/terraform/website/docs/d/runtimeconfig_config.html.markdown:23]: fixed

File: third_party/terraform/website/docs/d/runtimeconfig_config.html.markdown (hunk 2 above)
Comment (upodroid) [reply third_party/terraform/website/docs/d/runtimeconfig_config.html.markdown]: fixed

**Review by c2thorn — APPROVED** [review]

**Pull request merged**